earth_enterprise/src/fusion/autoingest/SConscript

```diff
@@ -189,6 +189,12 @@ for idlfile in idlfiles:
 
 commonobj = map(lambda src: env.SharedObject(src), commoncpp)
 
+for obj in commonobj:
+    env.Depends(obj, commonh)
+
+for cppfile in commoncpp:
+    env.Depends(cppfile[:-3] + '.obj', commonh)
```

Comment:
I believe the slice and append of '.obj' is not removing the existing '.' from the cppfile string.  From a quick test:

```>>> s = "filename.cpp"
>>> s[:-3]
'filename.'```

It might be better to use `os.path.splitext(cppfile)[0] + '.obj'`.